Terminal.Gui/ConsoleDrivers/FakeDriver/FakeDriver.cs

```diff
@@ -81,8 +81,8 @@ public override void Init (Action terminalResized)
 		FakeConsole.Clear ();
 		ResizeScreen ();
 		// Call InitializeColorSchemes before UpdateOffScreen as it references Colors
-		CurrentAttribute = MakeColor (Color.White, Color.Black);
-		InitializeColorSchemes ();
+		CurrentAttribute = new Attribute (Color.White, Color.Black);
+		//InitializeColorSchemes ();
 		ClearContents ();
```

Comment:
Theres a lot of commented out code in FakeDriver, maybe we should just delete it if it is no longer relevant?